pkg/controller/replica.go

```diff
@@ -137,24 +126,39 @@ func (r *ReconcileReplica) Reconcile(ctx context.Context, request reconcile.Requ
 	return reconcile.Result{}, nil
 }
 
-func (r *ReconcileReplica) triggerGarbageCollection(volumeName string) {
-	emptyCtx := context.Background()
-	deletionCtx, cancelFunc := context.WithCancel(emptyCtx)
+//nolint:contextcheck // context is not inherited by design
+func (r *ReconcileReplica) triggerManageReplica(ctx context.Context, volumeName string) {
+	manageReplicaCtx, w := workflow.New(context.Background(), workflow.WithDetails(consts.VolumeNameLabel, volumeName))
```

Comment:
Since `addToOperationQueue` creates workflow, is it necessary to also create one here?

Reply:
As there are several controllers that can trigger `manageReplica` workflow, I thought it would make a useful log to see which one the workflow came from.

Reply:
Yeah, I see that now.